cmd/kritis/admission/main.go

```diff
@@ -68,9 +68,7 @@ func main() {
 
 	kritisConfigs, err := kritisconfig.KritisConfigs("")
 	if err != nil {
-		errMsg := fmt.Sprintf("error getting kritis config: %v", err)
-		glog.Errorf(errMsg)
-		return
+		glog.Fatalf("failed to get kritis config: %v", err)
```

Comment:
As well as standard `log.Fatal`, `glog.Fatal[f]` calls `os.Exit` internally.
https://godoc.org/github.com/golang/glog#Fatal

Is it okay to replace existing `glog.Errorf` to `glog.Fatalf`?

Reply:
Yes, it's intentionally.
( The original code returns from `main` after `glog.Errorf`, which means it exits immediately. )

Reply:
👌